verticapy/tests/vDataFrame/test_vDF_plot_plotly.py

```diff
@@ -168,35 +168,42 @@ def acf_plot_result(load_plotly, amazon_vd):
 def regression_plot_result(load_plotly, dummy_scatter_vd):
     model = LinearRegression("LR_churn")
     model.fit(dummy_scatter_vd, ["x"], "y")
-    return model.plot()
```

Comment:
If we use return, the function will be called and will be entirely executed each time it is needed. In other words, the model will trained over and over. We should use yield instead which keeps the objects of the fixture for the entire of its scope. Then, we can make cleaning after the yield which would be when the fixture goes out of its scope.